modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/attachments_list.jspf

```diff
@@ -61,4 +61,59 @@
 		markupView="lexicon"
 		paginate="<%= paginate %>"
 	/>
-</liferay-ui:search-container>
\ No newline at end of file
+</liferay-ui:search-container>
+
+<%
+Date expirationDate = new Date(System.currentTimeMillis() + GetterUtil.getInteger(PropsUtil.get(PropsKeys.SESSION_TIMEOUT)) * Time.MINUTE);
+
+Ticket ticket = TicketLocalServiceUtil.addTicket(user.getCompanyId(), User.class.getName(), user.getUserId(), TicketConstants.TYPE_IMPERSONATE, null, expirationDate, new ServiceContext());
+%>
+
+<aui:script use="liferay-search-container">
```

Comment:
couldn't we add this to the WikiPortlet.es.js to make it reusable??

Reply:
Doing this reusable, it could be easily be used in page_attachment_action.jsp and edit_page_attachment.jsp with the liferay-ui:icon-delete taglib.

Reply:
Hey @robertoDiaz 

Is there a method to import 'liferay-search-container' in WikiPortlet.es.js ?

Reply:
Maybe this?? 

Layout.es.js:39

Reply:
Thanks, I'll try it.